src/Administration/Resources/app/administration/src/module/sw-flow/page/sw-flow-detail/index.js

```diff
@@ -305,13 +305,13 @@ export default {
             return State.commit('swFlowState/setFlow', flow);
         },
 
-        getDetailFlow() {
+        async getDetailFlow() {
             this.isLoading = true;
-            Shopware.State.dispatch('swFlowState/fetchTriggerActions');
```

Comment:
the easier solution would be to just await this request, but that would lead to slower load times as we do the requests sequentially and not in parallel

Reply:
I like it this way, yes of course just awaiting the promises after each other would be easier to read but we do that already in way to many places and in this case it really makes sense to do the two HTTP requests concurrently